docs/src/prototypes/popups/GridImagePicker/GridImagePicker.tsx

```diff
@@ -0,0 +1,71 @@
+import { Grid, Input, gridBehavior } from '@stardust-ui/react'
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import { ShorthandValue } from '../../../../../types/utils'
+import GridImagePickerItem, { GridPickerItemProps } from './GridImagePickerItem'
+
+export interface GridPickerProps {
+  as?: keyof React.ReactHTML
```

Comment:
Is there a strong reason why we support here just the HTML react elements?

Reply:
There is no strong reason for that. For now, it is just prototype + I wanted to constrain it to type, not to make ```as?: any``` as we have in Stardust components.

Reply:
Okay, let's leave it like this for now. When we will have all createComponent functionality available on the public API we will be able to reuse this from the Stardust library.